Miniscript-Template-000-Timelocks in Templates.mediawiki

```diff
@@ -11,71 +11,73 @@ Proposed Timelock Usage
+*after(2234779200) October 25th, 2040 Noon GMT
 
 
 ==Relative Epochtime based Timelocks==
 
-Proposed Examples of valid Block Height Relative Timelocks
-- Following [https://github.com/bitcoin/bips/blob/master/bip-0068.mediawiki BIP68], relative epoch timestamp timelocks can only go as far out as 33,554,431 seconds (1.06 years), as it is constrained by the same units as relative block height timelocks, 65,536 (2^16), where each unit represents 512 seconds of time (65,356 units * 512 seconds = 33,554,432 seconds).
+Proposed Examples of valid Block Height Relative Timelocks: 
+
+Background: 
+* Following [https://github.com/bitcoin/bips/blob/master/bip-0068.mediawiki BIP68], relative epoch timestamp timelocks can only go as far out as 33,554,431 seconds (1.06 years), as it is constrained by the same units as relative block height timelocks, 65,536 (2^16), where each unit represents 512 seconds (8 minutes and 32 seconds) of time (65,356 units * 512 seconds = 33,554,432 seconds).
 
 * An Epoch Timestamp is not valid until the network's MTP (Median Time Past) of the past 11 blocks is greater than the epoch timestamp, MTP is defined in: [https://github.com/bitcoin/bips/blob/master/bip-0113.mediawiki BIP113].
 
-* Since the 512 second incrimentor can not align exactly to a 24 hour day, for consistent recovery sake, relative epoch timestamps should be divisible by 100.
+* The smallest value that can be used to have the miniscript compiler interpret a relative timelock is older(4194305) (calculated by following the BIP68 spec of: 1 |= (1 << 22)). This is a 1 unit timelock of duration 512 seconds.
+
+* The maximum value for a relative epoch timelock is older(4259839) which is 65,535 unit timelock, resulting in 33,554,431 seconds, or 388 days.
 
-* The smallest value that can be used to have the miniscript compiler interpret a relative timelock is older(4194305) (calculated by following the BIP68 spec of: `1 |= (1 << 22)`). This is a 1 unit timelock, which is 512 seconds, or 8 minutes and 32 second timelock.
+* The 512 second incrementor is a common multiple for the amount of seconds in 4 days, which is 675 units. To encourage intuitive useage of relative timelocks, they should be multiples of 675.
+, an exception should be made for the largest possible epoch timestamp relative timelock.
 
-* The maximum value for a relative epoch timelock is `4259839` which is 65,535 unit timelock, resulting in 33,554,431 seconds, or 388 days, an exception should be made for the largest possible epoch timestamp relative timelock.
+'''Proposal''': Starting at 4194304, increment by 675 (4 days), for relative epochtime timelocks, the maximum value being 4259839.
 
-In practice, this makes each 100 unit incriment of a timelock being approximately ~14.2 hours in MTP.
 
 Propsoed Examples of valid Epoch Timestamp Absolute Timelocks
 
-*older(4194400)
-*older(4195000)
-*older(4200000)
-*older(4259839)
+*older(4194979) 4 days, minimum value
```

Comment:
In c2cc1af818d4a64e8de4d09794e5b6a4449964f4:

I believe this number should be 675, not 4194979.

Reply:
My goal was to show how in miniscript policy or output descriptors using miniscript the relative epoch timestamps would be represented. If you were to use `older(675)`, it would be interpreted as a 675 block height timelock.

To enforce a 4 day relative epochtime timelock, you take `(1 << 22)` to indicate an epoch timestamp relative timelock from BIP68 into decimal form, which is `4194304`. From there, you add 675 to do the 4 day relative timelock to get what I arrived to with `older(4194979)`.

I'm very open to presenting thsi information in a different way if you have a suggestion.

Reply:
Interesting. I actually had no idea that time-based relative timelocks worked this way (I'm not sure I ever really noticed that you could do time-based relative timelocks at all...)

I think as written this is fine. People will raise their eyebrows at it but they can check the relevant BIP or find this discussion.

Reply:
I had to do some digging in the`rust-minisript` tests after I spoke with @sanket1729 at Tabconf earlier this month. I agree that epoch time relative timelocks are the least likely used/known timelocks, I wanted to provide some reference documentation as a starting place.